utils/constants.jsx

```diff
@@ -1470,8 +1470,8 @@ export const Constants = {
     AUTOCOMPLETE_SPLIT_CHARACTERS: ['.', '-', '_'],
     ANIMATION_TIMEOUT: 1000,
     SEARCH_TIMEOUT_MILLISECONDS: 100,
-    DIAGNOSTICS_RUDDER_KEY: 'placeholder_rudder_key',
-    DIAGNOSTICS_RUDDER_DATAPLANE_URL: 'placeholder_rudder_dataplane_url',
+    DIAGNOSTICS_RUDDER_KEY: '1dodSrnEWiacrD76jx0nVToHNGu',
```

Comment:
This will also fail a test case, will be removing these when QA review is done and it should be fixed